src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -989,6 +1027,14 @@ export default class UniversalDiscoveryModule extends Component {
         return <span className="m-ud__location-not-allowed">{locationNotAllowedMessage}</span>;
     }
 
+    setMainContainerRef(ref) {
+        this._refMainContainer = ref;
```

Comment:
I would go for `React.createRef()`.

Reply:
But if we want to stick with this way in order to look it the same as in other places, then OK.